tiled/readers/tiff.py

```diff
@@ -49,7 +49,10 @@ def microstructure(self):
         return MachineDataType.from_numpy_dtype(self._file.series[0].dtype)
 
     def macrostructure(self):
-        shape = tuple(self._file.shaped_metadata[0]["shape"])
+        if self._file.is_shaped:
+            shape = tuple(self._file.shaped_metadata[0]["shape"])
+        else:
+            shape=[len(self._file.asarray()), len(self._file.asarray()[0])]
```

Comment:
Two notes on this line:

1. We need `shape` to be a `tuple`.
2. Does `self._file.asarray().shape` work? What is `type(self._file.asarray())`?

Reply:
I confirmed that this function returns a NumPy array and proceed to add the proper command using `self._file.asarray().shape`